src/DynamoCoreWpf/Views/GuidedTour/GuideBackground.xaml.cs

```diff
@@ -33,14 +36,33 @@ public double CanvasHeight
         }
 
         /// <summary>
-        /// Rect with the size of the Dynamo Window 
+        /// Rect with the size of the Dynamo Window regularly updating its size depending the window's size. Those are represented by 
+        /// SystemParameters.PrimaryScreenWidth and SystemParameters.PrimaryScreenHeight
         /// </summary>
-        public Rect WindowsRect { get { return new Rect(0, 0, CanvasWidth, CanvasHeight); } }
+        public Rect WindowsRect { get { return new Rect(0, 0, SystemParameters.PrimaryScreenWidth, SystemParameters.PrimaryScreenHeight); } }
 
         /// <summary>
         /// Rect used to cut the hole on the guide background 
         /// </summary>
-        public Rect HoleRect { get { return new Rect (50, 100, 250, 130); } }
+        public Rect HoleRect {
+            get {
+                return hole; 
+            }
+            set { 
+                hole = value;
+                NotifyPropertyChanged("HoleRect");
```

Comment:
please use `nameof(HoleRect)` instead of hardcoded string

Reply:
Also cant we just use `RaisePropertyChanged()`?

Reply:
Fixed the hardcoded string as you suggested, but the RaisePropertyChanged didnt work for me :/